WordPress/Classes/Services/PostService.swift

```diff
@@ -0,0 +1,36 @@
+extension PostService {
```

Comment:
Probably let's name this file `PostService+RefreshStatus` or `PostService+Async` or `PostService+AppResume` or something?